app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -0,0 +1,29 @@
+@import "twitter-bootstrap-static/bootstrap";
+@import "twitter-bootstrap-static/fontawesome";
+
+body {
+  background: #f3f4f5;
+}
+
+// Make navigation & page header background white
+.navbar-default,
+._page-header {
+  background-color: #FFF;
```

Comment:
Color literals like `#FFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFF` should be written as `#fff`